var/spack/repos/builtin/packages/fjcontrib/package.py

```diff
@@ -0,0 +1,84 @@
+    version('1.020', sha256='1ec2041cb5e7f050e20823a31111c7e61d24c4d7ee546c9714861b16eca83a04')
+    version('1.019', sha256='adec9e1d3c48fec8d6786b64d9c4e95e79129a156f0b0ed3df7e98d57882ffb2')
+    version('1.018', sha256='fe9d9661cefba048b19002abec7e75385ed73a8a3f45a29718e110beab142c2c')
+    version('1.017', sha256='6f711b8b7f63a9a1bdf83cc1df6f7785bd3394e96dddba12186dfb5de352362c')
+    version('1.016', sha256='9301027a71098ef75c37742a2f60631b2e39eb4d6b73b18db48d31e6aed830a0')
+    version('1.015', sha256='4b9d56d2b6ae56a894b64920ac9ef59013b1d0a54454e716dd98e724f9ea82c0')
+    version('1.014', sha256='c8de7a0cc19be7684dcfbc3101f9cd4c9426daa3009726acd5a5cbe5e699544b')
+    version('1.013', sha256='c2b4838a41431cd1c28d3a836f10c00bb84eb6eb2e6586a28badab66518107a6')
+    version('1.012', sha256='9215ee5b6465fa64296a0139e727a2eb61255d50faccafaa5b241e408ae54821')
+    version('1.011', sha256='7e4e62755d5905e9dc4a6ef29728c03da538cbbff381eed32a35f06934304b1d')
+    version('1.010', sha256='1ae74ef44e3d8ce96fd7eb61571819e8891ff708fd468a71780e693c2081176c')
+    version('1.009', sha256='dc12cce6f0468b4fab9ec4d9fbb2c3d76e942b3b6965cf609963f65c69c08d2f')
+    version('1.008', sha256='8cc98d837b0579b605a4731fb69882084948e1f1d2ee20af5c8adade2434e4ed')
+    version('1.007', sha256='70b9d57e5c690d9af8b5be96eefeea9f8c4c4d1a7f000fe1dac30e5aae36077f')
+    version('1.006', sha256='e2b6d2c5a666eb4db6bef89ea176df4bc1cdec316f8d18594ab54eb0ce8dcbb6')
+    version('1.005', sha256='265e682a05c1f5e5cf1560cc4efa99e07b211cb2add5f3a09b5be4363ab9cc7f')
+    version('1.004', sha256='b397e6f822a1d539ac1f63dc8c0709a7a00eed25f4c011114be71f0b38f8f514')
+    version('1.003', sha256='79b2d42b1b7a99686fa2e0d95ad39050fd974fdf5ea5066271185ae308299b72')
+    version('1.002', sha256='b48d2cc0a9c2fcfe3bb7faee3217a50191be4150a62ed2f6419fd0025d2cec92')
+    version('1.001', sha256='6030dfee59040d2ada4ee582254e60b9ecf4b62598129e7b0a114672cf267491')
+    version('1.000', sha256='95fa36ae48f03cb941d632b0a537995fb7148a6bd028c4978fab8b7b04332c3b')
+    version('0.001', sha256='51f24ad55e28fb1f9d698270602e5077c920fcf58d8ccfd274efbe829d7dd821')
+    version('0.000', sha256='9486b11201e6b6e181b8a3abecd929403ae9aa67de0eb8b7353fb82ab4b89f41')
+
+    depends_on('fastjet')
+
+    build_targets = ['all', 'fragile-shared']
+    install_targets = ['install', 'fragile-shared-install']
+
+    def configure_args(self):
+        args = ['--fastjet-config=' +
+                self.spec['fastjet'].prefix.bin +
+                '/fastjet-config',
+                "CXXFLAGS=-O3 -Wall -g -fPIC"]
+        return args
+
+    def build(self, spec, prefix):
+        with working_dir(self.build_directory):
+            for target in self.build_targets:
+                inspect.getmodule(self).make(target)
+
+    def install(self, spec, prefix):
+        with working_dir(self.build_directory):
+            for target in self.install_targets:
+                inspect.getmodule(self).make(target)
```

Comment:
This seems to be the default for AutotoolsPackage, although the base class will run `make('install', 'fragile-shared-install')`, not sure if it needs to be run sequentially.

Reply:
Yes, it has to be ran sequentially.

Reply:
Hmm, I wonder if we should change this in the `AutotoolsPackage` base class then... @alalazo any thoughts?